environment/src/test/java/com/sequenceiq/environment/parameters/validation/validators/AwsParameterValidatorTest.java

```diff
@@ -92,6 +95,53 @@ void validateAndDetermineAwsParametersAttached() {
         verify(noSqlTableCreationModeDeterminerService, never()).determineCreationMode(any(), any());
     }
 
+    @Test
+    void validateNoS3GuardCheckOnUpdateAwsDiskEncryptionParameters() {
+        AwsParametersDto awsParameters = AwsParametersDto.builder()
+                .withDynamoDbTableName("tablename")
+                .build();
+        ParametersDto parametersDto = ParametersDto.builder()
+                .withAwsParameters(awsParameters)
+                .build();
+        EnvironmentDto environmentDto = new AwsParameterValidatorTest.EnvironmentDtoBuilder()
+                .withAwsParameters(AwsParametersDto.builder()
+                        .build())
+                        .build();
```

Comment:
I suppose `environmentDto` should have the same DynamoDB table `"tablename"` set for consistency, as that would be the realistic scenario.

Reply:
Addressed this!